### DISCUSSION TITLE
This post shows up still in 2019 🔢

### DISCUSSION THREAD

**Discussion by bswatantra** [jenssegers/laravel-mongodb#2483] [Q&A]
This post shows up still in 2019 🔢 

`use MongoDB\BSON\UTCDateTime;`

`new UTCDateTime(new \DateTime('now'));`

Carbon extends DateTime so a carbon instance should be usable:

`new UTCDateTime(Carbon::now());`

_Originally posted by @marcnewton in https://github.com/jenssegers/laravel-mongodb/issues/594#issuecomment-453159291_

**Comment by bswatantra** [comment]
while fetching data I get 
```
"created_at": {
        "$date": {
          "$numberLong": "1653035305018"
        }
      }
```

but I want it 
```
2022-05-20T08:28:25.021000Z
OR
022-05-20 14:13:27
```

**Reply by Giacomo92** [reply]
Hi @bswatantra,
I think this issue is not related to this repo. I suggest you to always save dates to database like this:
`new UTCDateTime(new DateTime($mydate));`
I think it is the only way to retrieve a correct instance of date.
